src/components/PopupDeleteConfirm.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+    this._popupForm = this._popup.querySelector(".modal__form");
+    this._submitButton = this._popupForm.querySelector(".modal__button");
+  }
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.